wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-    1. Manifests for a set of specific common services:
-        - Istio
-        - KNative
-        - Dex
-        - Cert-Manager
-
-#### Cross-cutting and Externally Facing Processes
-
-##### With Application Owners
-
-- Aid applications owners in creating kustomize manifests for their application,
-  inside the app repo, if those don't exist already.
-- Communicate with application owners to agree upon the version they want to be
-  included in the next Kubeflow release.
-
-##### With Distribution Owners
-
-- Coordinate with distribution owners, to make sure they are in-sync about the
-  release schedule and have time to test and bring their distributions
-  up-to-date.
-
-### Out of scope
-
-This WG is NOT going to:
-- Maintain deployment-specific tools like `kfctl`.
-- Maintain distribution-specific manifests.
-- Decide which applications to include in Kubeflow.
-- Decide which variant of an application to include (e.g., KFP Standalone vs
-  KFP with Istio).
-- Create and maintain one or more Kubeflow distributions.
-- Support configurations with environment-specific requirements, like special
-  hardware, different versions of third-party apps (e.g., Istio, KNative, etc.)
-  or custom OIDC providers.
-- Support and promote a specific deployment tool (e.g., `kfctl`). Opinionated
-  deployment tools can extend the base kustomizations to create manifests that
-  support their methods.
-    - For example, people invested in `kfctl` can create overlays that enable
-      the use of `kfctl`'s parameter substitution, which expects a specific
-      folder structure (`params.env`).
+- Enable users / distributions to install, extend and maintain Kubeflow as a multi-tenant platform for multiple users
+- This includes dependencies, security efforts and exemplary integration with popular tools and frameworks.
+- Synchronize the manifests (Helm, Kustomize) between working groups
+- We try to be compatible with the popular Kubernetes clusters (Kind, Rancher, AKS, EKS, GKE, ...)
+- **We do not support a specific deployment tool (e.g., ArgoCD, Flux)**
+- The default installation shall not contain deep integration with external cloud services or closed source solutions, instead we aim for Kubernetes-native solutions and light authentication and authorization integration with external IDPs
+- We provide hints and experimental examples how a user / distribution could integrate non-default external authentication (e.g. companies Identity Provider) and popular non-default services on his own
+- There is the evolving and not exhaustive list of dependencies for a proper multi-tenant platform installation: Istio, KNative, Dex, Oauth2-proxy, Cert-Manager, ...
+- There is the evolving and not exhaustive list of applications:  KFP, Trainer, Dashboard, Workspaces / Noteboks, Kserve, Spark, ...
+
+## Communication Tasks
+
+### With Application Owners
+
+- Aid the application owner in creating manifests (Helm, Kustomize) for his application
```

Comment:
I'm not sure that requiring the manifests WG to support the upstream manifests is sustainable.

But obviously, it is something that the individuals who are participating might also choose to do if they are so inclined.

Reply:
Yes, you always have to keep in mind that we are volunteers. All of this is best-effort. We try it. Sometimes other working groups need help to understand for example securitycontexts of a pod, since they are rather focused on the source code. Or we help them to fix the kustomize 5 warnings.

Reply:
Over the last five years it was very sustainable and we made great progress.